tests/testcases/test_paths.py

```diff
@@ -36,15 +35,15 @@ def tearDown(self):
         quick_cleanup()
 
     def test_touch(self):
-        touch("/tmp/a/b/c/d/e")
-        self.assertTrue(os.path.exists("/tmp/a/b/c/d/e"))
-        self.assertTrue(os.path.isfile("/tmp/a/b/c/d/e"))
-        self.assertRaises(ValueError, lambda: touch("/tmp/a/b/c/d/f/"))
+        touch(tmp + "/a/b/c/d/e")
+        self.assertTrue(os.path.exists(tmp + "/a/b/c/d/e"))
+        self.assertTrue(os.path.isfile(tmp + "/a/b/c/d/e"))
+        self.assertRaises(ValueError, lambda: touch(tmp + "/a/b/c/d/f/"))
 
     def test_mkdir(self):
-        mkdir("/tmp/b/c/d/e")
-        self.assertTrue(os.path.exists("/tmp/b/c/d/e"))
-        self.assertTrue(os.path.isdir("/tmp/b/c/d/e"))
+        mkdir(tmp + "/b/c/d/e")
+        self.assertTrue(os.path.exists(tmp + "/b/c/d/e"))
+        self.assertTrue(os.path.isdir(tmp + "/b/c/d/e"))
```

Comment:
```suggestion
        self.assertTrue(os.path.isdir(os.path.join(tmp, "/b/c/d/e")))
```

here and everywhere else

Reply:
If you say you'd rather keep the string concatenation in those cases I'm fine with it